tools/build/third_party/glslang.BUILD

```diff
@@ -44,7 +44,9 @@ cc_library(
     copts = cc_copts() + [
         "-DNV_EXTENSIONS",
         "-Wno-unused-variable",
-    ],
+    ] + select({
+        "@gapid//tools/build:linux": ["-Wno-error=class-memaccess"],
```

Comment:
Can you:
1. create a GAPID issue to track that glslang should fix their warnings
2. Add a TODO referring to this issue